docs/cli.md

```diff
@@ -92,6 +92,18 @@ Owned OwnerCap object must be passed to this command for authorization.
 This command requires that a wallet is connected to the CLI...
 {% endhint %}
 
+---
+
+**`nexus tool set-invocation-cost --tool-fqn <fqn> --owner-cap <object_id> --invocation-cost <mist>`**
+
+Tool owners can change the invocation cost of their Tools specified by the FQN. This operation requires that the `OwnerCap<OverGas>` object is passed to the command and owned by the transaction sender.
```

Comment:
```suggestion
Tool owners can change the invocation cost of their Tools specified by the FQN. This operation requires that the `OwnerCap<OverGas>` object (which will be owned by the owner of the tool) is passed to the command and owned by the transaction sender.
```
I'd make it even more explicit that this ownercap essentially means you have tool owner rights. (Unless I'm wrong and it's more nuanced?)

Reply:
You can create an `OwnerCap<OverGas>` from an `OwnerCap<OverTool>` freely but the `OverGas` one can only be used to authorize gas-related calls.

Reply:
So practical translation: it's not strictly a tool owner's action but either: 
a) a tool owner or 
b) anyone who's been delegated to manage gas-related calls BY the tool owner.

My point was we should probably document this relationship better, either here or in the gas service doc in nexus-next. 

I feel the latter might make more sense and then we reference in this doc. If you agree @kouks feel free to leave this as a TODO with issue: https://github.com/Talus-Network/nexus-next/issues/283)

Reply:
yeah owner caps are owned objects that the owner can freely transfer, sounds good, I'll add the TODO

Reply:
https://github.com/Talus-Network/nexus-sdk/pull/162/commits/9a2d680d99e35776b29fe02075f469a34cf125bd